container_pipeline/management/commands/triggerbuilds.py

```diff
@@ -6,7 +6,8 @@
 from django.utils import timezone
 from django.conf import settings
 
-from container_pipeline.models.tracking import ContainerImage, Package
```

Comment:
this should have just worked too!

Reply:
True, i just refactored the placement based on specialisation of the classes, so it makes sense programatically. I think ContainerImage can be used to store more than just repo tracking information.